Sources/_CFURLSessionInterface/include/CFURLSessionInterface.h

```diff
@@ -34,6 +34,27 @@
 #include <winsock2.h>
 #endif
 
+// 7.84.0 or later
+#if LIBCURL_VERSION_MAJOR > 7 || (LIBCURL_VERSION_MAJOR == 7 && LIBCURL_VERSION_MINOR > 84) || (LIBCURL_VERSION_MAJOR == 7 && LIBCURL_VERSION_MINOR == 84 && LIBCURL_VERSION_PATCH >= 0)
+#define NS_CURL_CURLINFO_CAINFO_SUPPORTED 1
```

Comment:
I inverted the sense from "missing" to "supported" because it was a lot easier to understand.